kpi/templates/base_simple.html

```diff
@@ -2,7 +2,7 @@
 
 {% load static %}
 <!doctype html>
-<html>
+<html dir="auto">
```

Comment:
Does this line do anything? As you noted in the PR description, `dir="auto"` doesn't appear to transfer down to child elements. I can't find a difference on `/accounts/login/` - the layout, text direction, and input text direction appear to all stay the same as on `beta`.

Reply:
I thought that although it doesn't transfer down, it does something sometimes :P I tried to find any documentation or SO question that would confirm that, but there is nothing. I removed it